src/components/Container.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { theme } from '../constants';
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file and about the style files.